apps/web/app/team/[slug]/[type]/page.tsx

```diff
@@ -27,12 +27,33 @@ export const generateMetadata = async ({ params, searchParams }: PageProps) => {
   });
 
   const profileName = event?.profile?.name ?? "";
+  const profileImage = event?.profile.image;
   const title = event?.title ?? "";
-
-  return await _generateMetadata(
+  const meeting = {
+    title,
+    profile: { name: profileName, image: profileImage },
+    users: [
+      ...(event?.users || []).map((user) => ({
+        name: `${user.name}`,
+        username: `${user.username}`,
+      })),
+    ],
+  };
+  const metadata = await generateMeetingMetadata(
+    meeting,
     (t) => `${booking?.uid && !!booking ? t("reschedule") : ""} ${title} | ${profileName}`,
-    (t) => `${booking?.uid ? t("reschedule") : ""} ${title}`
+    (t) => `${booking?.uid ? t("reschedule") : ""} ${title}`,
+    isBrandingHidden,
+    getOrgFullOrigin(eventData.entity.orgSlug ?? null)
   );
+
+  return {
+    ...metadata,
+    robots: {
+      follow: !(event?.hidden || !isSEOIndexable),
+      index: !(event?.hidden || !isSEOIndexable),
```

Comment:
- `noindex` and `nofollow` are not supported in App Router. We need to use `follow` and `index` instead.